tests/acceptance/features/apiSharingNg/shareInvitations.feature

```diff
@@ -774,3 +774,70 @@ Feature: Send a sharing invitations
       | Co Owner        | folder        | FolderToShare  |
       | Uploader        | folder        | FolderToShare  |
       | Manager         | folder        | FolderToShare  |
+
+
+  Scenario Outline: try to send sharing invitation to multiple groups
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Carol    |
+      | Bob      |
+    And group "grp1" has been created
+    And group "grp2" has been created
+    And the following users have been added to the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+      | Carol    | grp2      |
+      | Bob      | grp2      |
+    And user "Alice" has uploaded file with content "to share" to "/textfile1.txt"
+    And user "Alice" has created folder "FolderToShare"
+    When user "Alice" sends the following share invitation using the Graph API:
+      | resourceType    | <resource-type>    |
+      | resource        | <path>             |
+      | space           | Personal           |
+      | sharee          | grp1, grp2         |
+      | shareType       | group, group       |
```

Comment:
```suggestion
      | shareType       | group       |
```

Reply:
It is added just to differentiate between share type, for instance, if the resource is shared to group and user at once.